wolfSSL/src/chapter02.md

```diff
@@ -742,6 +1610,22 @@ Enables 1024-bit test certificate and key buffers located in `<wolfssl_root>/wol
 
 Enables 2048-bit test certificate and key buffers located in `<wolfssl_root>/wolfssl/certs_test.h`. Helpful when testing on and porting to embedded systems with no filesystem.
 
+#### USE_CERT_BUFFERS_3072
+
+Enables 3072-bit test certificate and key buffers located in `<wolfssl_root>/wolfssl/certs_test.h`. Helpful when testing on and porting to embedded systems with no filesystem.
+
+#### USE_CERT_BUFFERS_4096
+
+Enables 3072-bit test certificate and key buffers located in `<wolfssl_root>/wolfssl/certs_test.h`. Helpful when testing on and porting to embedded systems with no filesystem.
+
+#### USE_CERT_BUFFERS_25519
+
+Enables Ed25519 test certificate and key buffers located in `<wolfssl_root>/wolfssl/certs_test.h`. Helpful when testing on and porting to embedded systems with no filesystem.
+
+#### USE_WOLFSSL_IO
+
+Enables the wolfSSL IO functions.
```

Comment:
Please expand details. This is a very important macro. It enables callbacks for send/recv. Please provide an example or link to https://github.com/wolfSSL/wolfssl-examples/blob/master/tls/client-tls-callback.c